### PR TITLE
Adjustments lp

### PR DESCRIPTION
**Pull request by JbellonDev**
Adjustments completed according document. Notifications created as Layout file to be edited.

### CODE REVIEW

File: dashboard.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline dashboard.html]: remover espacios!

File: documents.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline documents.html]: espacios!

File: layout.html [hunk 3]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline layout.html]: espacio!

File: milestones.html [hunk 4]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline milestones.html]: :S

File: notes.html [hunk 5]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline notes.html]: same

File: services.html [hunk 6]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline services.html]: !!!

File: to-do.html [hunk 7]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline to-do.html]: !!!

File: your-team.html [hunk 8]
```diff
@@ -23,7 +23,7 @@
           <!-- company logo -->
           <div class="company-logo-container">
             <img id="company-logo" alt="Company Logo" src="">
-            <img id="company-logo-mini" alt="Company Logo" src="">
+
```
Comment (hrodriguezimagina) [inline your-team.html]: !!!

**Review by hrodriguezimagina — CHANGES_REQUESTED** [review]
porfa remover los espacios

**Comment by JbellonDev** [comment]
Espacios removidos.

**Pull request merged**